examples/rsp-next-ts/pages/_document.tsx

```diff
@@ -0,0 +1,15 @@
+import { Html, Head, Main, NextScript } from 'next/document'
+import {LocalizedStringProvider} from '@adobe/react-spectrum/i18n';
+
+export default function Document(props) {
+  return (
+    <Html lang="en">
+      <Head />
+      <body>
+        <Main />
+        <LocalizedStringProvider locale="en" />
```

Comment:
One thing to consider is what to do if you had both RSP and RAC in your app since there is overlap between the two in terms of translations, and the script tags would overwrite each other. You might need something more manual using `PackageLocalizationProvider` in that case.